example.py

```diff
@@ -1,18 +1,18 @@
 import gym_electric_motor as gem
-from gym_electric_motor import reference_generators as rg
 import gem_controllers as gc
 from gym_electric_motor.visualization import MotorDashboard
 from gym_electric_motor.physical_system_wrappers import FluxObserver
 
 
-env_id = 'Cont-CC-SCIM-v0'
+if __name__ == '__main__':
```

Comment:
"env_id" got deleted and I think everything after the if clause should also be indented.